The job **`x86_64-gnu-tools`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/9885253032/job/27302965495) [(plain)](https://github.com/rust-lang/rust/commit/de88bc5c8921774681ecac0a5249b370a9768a6c/checks/27302965495/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
  |                 ^ unclosed delimiter
4 |     } else {
  |     ^ mismatched closing delimiter

.....error: `~const` trait not allowed with `!` trait polarity modifier
  |
  |
9 |     i32: ~const !Copy,
  |          |
  |          |
  |          there is not a well-defined meaning for a `~const !` trait

..error: `~const` trait not allowed with `?` trait polarity modifier
    |
    |
149 | struct S<T:~  const   ?  Sized>(std::marker::PhantomData<T>);
    |            |
    |            |
    |            there is not a well-defined meaning for a `~const ?` trait

test::system_tests ... F
test::system_tests ... F
error: `~const` trait not allowed with `?` trait polarity modifier
##[error]   --> tests/target/type.rs:156:20
    |
156 | struct S<T: ~const ?Sized>(std::marker::PhantomData<T>);
    |             |
    |             |
    |             there is not a well-defined meaning for a `~const ?` trait
 176/177

test::idempotence_tests ... F



failures:

---- test::system_tests stdout ----
2024-07-11T04:27:14.993506Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:14.996187Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.000674Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.026952Z  WARN rustfmt_nightly::test: Default value 2015 used explicitly for edition
2024-07-11T04:27:15.035310Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.086894Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_macro_bodies
2024-07-11T04:27:15.094063Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.111058Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_modules
2024-07-11T04:27:15.131733Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_imports
2024-07-11T04:27:15.234470Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.305040Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.313998Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.326113Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_imports
2024-07-11T04:27:15.330965Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.334714Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.353525Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.358258Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.382174Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.383170Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.385049Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.385054Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.505369Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.524591Z  WARN rustfmt_nightly::test: Default value true used explicitly for match_arm_blocks
2024-07-11T04:27:15.525518Z  WARN rustfmt_nightly::test: Default value Vertical used explicitly for trailing_comma
2024-07-11T04:27:15.528381Z  WARN rustfmt_nightly::test: Default value false used explicitly for match_block_trailing_comma
2024-07-11T04:27:15.531120Z  WARN rustfmt_nightly::test: Default value AlwaysSameLine used explicitly for control_brace_style
2024-07-11T04:27:15.533851Z  WARN rustfmt_nightly::test: Default value true used explicitly for empty_item_single_line
2024-07-11T04:27:15.535633Z  WARN rustfmt_nightly::test: Default value SameLineWhere used explicitly for brace_style
2024-07-11T04:27:15.539205Z  WARN rustfmt_nightly::test: Default value SameLineWhere used explicitly for brace_style
2024-07-11T04:27:15.541071Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.541994Z  WARN rustfmt_nightly::test: Default value 80 used explicitly for comment_width
2024-07-11T04:27:15.544941Z  WARN rustfmt_nightly::test: Default value Default used explicitly for use_small_heuristics
2024-07-11T04:27:15.546931Z  WARN rustfmt_nightly::test: Default value false used explicitly for fn_single_line
2024-07-11T04:27:15.548641Z  WARN rustfmt_nightly::test: Default value false used explicitly for disable_all_formatting
2024-07-11T04:27:15.550271Z  WARN rustfmt_nightly::test: Default value true used explicitly for remove_nested_parens
2024-07-11T04:27:15.552016Z  WARN rustfmt_nightly::test: Default value true used explicitly for force_explicit_abi
2024-07-11T04:27:15.552882Z  WARN rustfmt_nightly::test: Default value false used explicitly for format_strings
2024-07-11T04:27:15.552887Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.554847Z  WARN rustfmt_nightly::test: Default value 10 used explicitly for short_array_element_width_threshold
2024-07-11T04:27:15.558669Z  WARN rustfmt_nightly::test: Default value 100 used explicitly for doc_comment_code_block_width
2024-07-11T04:27:15.559765Z  WARN rustfmt_nightly::test: Default value 100 used explicitly for doc_comment_code_block_width
2024-07-11T04:27:15.562815Z  WARN rustfmt_nightly::test: Default value Tall used explicitly for fn_params_layout
2024-07-11T04:27:15.565110Z  WARN rustfmt_nightly::test: Default value 50 used explicitly for single_line_let_else_max_width
2024-07-11T04:27:15.567810Z  WARN rustfmt_nightly::test: Default value Mixed used explicitly for imports_layout
2024-07-11T04:27:15.567814Z  WARN rustfmt_nightly::test: Default value Block used explicitly for imports_indent
2024-07-11T04:27:15.569587Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_generated_files
2024-07-11T04:27:15.572967Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_generated_files
2024-07-11T04:27:15.573838Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_generated_files
2024-07-11T04:27:15.577791Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_modules
2024-07-11T04:27:15.579487Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.579492Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.581342Z  WARN rustfmt_nightly::test: Default value false used explicitly for force_multiline_blocks
2024-07-11T04:27:15.583448Z  WARN rustfmt_nightly::test: Default value false used explicitly for use_try_shorthand
2024-07-11T04:27:15.585240Z  WARN rustfmt_nightly::test: Default value false used explicitly for spaces_around_ranges
2024-07-11T04:27:15.587519Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.590802Z  WARN rustfmt_nightly::test: Default value false used explicitly for reorder_impl_items
2024-07-11T04:27:15.593582Z  WARN rustfmt_nightly::test: Default value Wide used explicitly for type_punctuation_density
2024-07-11T04:27:15.594584Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.597148Z  WARN rustfmt_nightly::test: Default value true used explicitly for merge_derives
2024-07-11T04:27:15.598204Z  WARN rustfmt_nightly::test: Default value false used explicitly for format_macro_matchers
2024-07-11T04:27:15.605147Z  WARN rustfmt_nightly::test: Default value false used explicitly for normalize_comments
2024-07-11T04:27:15.608978Z  WARN rustfmt_nightly::test: Default value Never used explicitly for match_arm_leading_pipes
2024-07-11T04:27:15.610861Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_macro_bodies
2024-07-11T04:27:15.622402Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.623351Z  WARN rustfmt_nightly::test: Default value 4 used explicitly for tab_spaces
2024-07-11T04:27:15.623355Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.625177Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.627023Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.627981Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.628857Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.633256Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.635163Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.635168Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.636078Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.636081Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.636083Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.636969Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.637885Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.640497Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.641468Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.643941Z  WARN rustfmt_nightly::test: Default value false used explicitly for normalize_doc_attributes
2024-07-11T04:27:15.645746Z  WARN rustfmt_nightly::test: Default value false used explicitly for hard_tabs
2024-07-11T04:27:15.648233Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_imports
2024-07-11T04:27:15.649143Z  WARN rustfmt_nightly::test: Default value false used explicitly for condense_wildcard_suffixes
2024-07-11T04:27:15.650959Z  WARN rustfmt_nightly::test: Default value false used explicitly for use_field_init_shorthand
Warning: the `merge_imports` option is deprecated. Use `imports_granularity="Crate"` instead
2024-07-11T04:27:15.673848Z  WARN rustfmt_nightly::test: Default value true used explicitly for merge_derives
2024-07-11T04:27:15.686484Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
Warning: the `fn_args_layout` option is deprecated. Use `fn_params_layout`. instead
2024-07-11T04:27:15.743244Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_imports
2024-07-11T04:27:15.763648Z  WARN rustfmt_nightly::test: Default value [] used explicitly for skip_macro_invocations
2024-07-11T04:27:15.787456Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.787462Z  WARN rustfmt_nightly::test: Default value Block used explicitly for imports_indent
thread '<unnamed>' panicked at src/tools/rustfmt/src/test/mod.rs:189:9:
assertion `left == right` failed: 1 system tests failed
  left: 1
 right: 0
 right: 0
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'test::system_tests' panicked at src/tools/rustfmt/src/test/mod.rs:76:10:
Failed to join a test thread: Any { .. }

---- test::idempotence_tests stdout ----
2024-07-11T04:27:14.982282Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:14.991832Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:14.996080Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.004485Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.045477Z  WARN rustfmt_nightly::test: Default value 2015 used explicitly for edition
2024-07-11T04:27:15.057315Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.092644Z  WARN rustfmt_nightly::test: Default value false used explicitly for format_macro_matchers
2024-07-11T04:27:15.095479Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.096310Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.099240Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.101505Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.102382Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.110553Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.111424Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.132736Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_macro_bodies
2024-07-11T04:27:15.140152Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.161796Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_modules
2024-07-11T04:27:15.195789Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_imports
2024-07-11T04:27:15.309130Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.318423Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.320172Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.400079Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.408817Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.423739Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_imports
2024-07-11T04:27:15.429049Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.433749Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.459772Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.466784Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.495331Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.496320Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.498084Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.498090Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.617333Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.652368Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.678448Z  WARN rustfmt_nightly::test: Default value true used explicitly for match_arm_blocks
2024-07-11T04:27:15.679356Z  WARN rustfmt_nightly::test: Default value Vertical used explicitly for trailing_comma
2024-07-11T04:27:15.682168Z  WARN rustfmt_nightly::test: Default value false used explicitly for match_block_trailing_comma
2024-07-11T04:27:15.684931Z  WARN rustfmt_nightly::test: Default value AlwaysSameLine used explicitly for control_brace_style
2024-07-11T04:27:15.687698Z  WARN rustfmt_nightly::test: Default value true used explicitly for empty_item_single_line
2024-07-11T04:27:15.689448Z  WARN rustfmt_nightly::test: Default value SameLineWhere used explicitly for brace_style
2024-07-11T04:27:15.693029Z  WARN rustfmt_nightly::test: Default value SameLineWhere used explicitly for brace_style
2024-07-11T04:27:15.694841Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.695669Z  WARN rustfmt_nightly::test: Default value 80 used explicitly for comment_width
2024-07-11T04:27:15.698560Z  WARN rustfmt_nightly::test: Default value Default used explicitly for use_small_heuristics
2024-07-11T04:27:15.700505Z  WARN rustfmt_nightly::test: Default value false used explicitly for fn_single_line
2024-07-11T04:27:15.702274Z  WARN rustfmt_nightly::test: Default value false used explicitly for disable_all_formatting
2024-07-11T04:27:15.703955Z  WARN rustfmt_nightly::test: Default value true used explicitly for remove_nested_parens
2024-07-11T04:27:15.705627Z  WARN rustfmt_nightly::test: Default value true used explicitly for force_explicit_abi
2024-07-11T04:27:15.706476Z  WARN rustfmt_nightly::test: Default value false used explicitly for format_strings
2024-07-11T04:27:15.706480Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.708413Z  WARN rustfmt_nightly::test: Default value 10 used explicitly for short_array_element_width_threshold
2024-07-11T04:27:15.712323Z  WARN rustfmt_nightly::test: Default value 100 used explicitly for doc_comment_code_block_width
2024-07-11T04:27:15.713475Z  WARN rustfmt_nightly::test: Default value 100 used explicitly for doc_comment_code_block_width
2024-07-11T04:27:15.716508Z  WARN rustfmt_nightly::test: Default value Tall used explicitly for fn_params_layout
2024-07-11T04:27:15.718796Z  WARN rustfmt_nightly::test: Default value 50 used explicitly for single_line_let_else_max_width
2024-07-11T04:27:15.721408Z  WARN rustfmt_nightly::test: Default value Block used explicitly for imports_indent
2024-07-11T04:27:15.722357Z  WARN rustfmt_nightly::test: Default value Block used explicitly for imports_indent
2024-07-11T04:27:15.722361Z  WARN rustfmt_nightly::test: Default value Mixed used explicitly for imports_layout
2024-07-11T04:27:15.723246Z  WARN rustfmt_nightly::test: Default value Mixed used explicitly for imports_layout
2024-07-11T04:27:15.723250Z  WARN rustfmt_nightly::test: Default value Block used explicitly for imports_indent
2024-07-11T04:27:15.724965Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_generated_files
2024-07-11T04:27:15.728279Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_generated_files
2024-07-11T04:27:15.729130Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_generated_files
2024-07-11T04:27:15.733092Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_modules
2024-07-11T04:27:15.734735Z  WARN rustfmt_nightly::test: Default value false used explicitly for wrap_comments
2024-07-11T04:27:15.734741Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.736454Z  WARN rustfmt_nightly::test: Default value false used explicitly for force_multiline_blocks
2024-07-11T04:27:15.739629Z  WARN rustfmt_nightly::test: Default value true used explicitly for trailing_semicolon
2024-07-11T04:27:15.740578Z  WARN rustfmt_nightly::test: Default value false used explicitly for use_try_shorthand
2024-07-11T04:27:15.742394Z  WARN rustfmt_nightly::test: Default value false used explicitly for spaces_around_ranges
2024-07-11T04:27:15.744590Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.747205Z  WARN rustfmt_nightly::test: Default value false used explicitly for reorder_impl_items
2024-07-11T04:27:15.750059Z  WARN rustfmt_nightly::test: Default value Wide used explicitly for type_punctuation_density
2024-07-11T04:27:15.751032Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.753478Z  WARN rustfmt_nightly::test: Default value true used explicitly for merge_derives
2024-07-11T04:27:15.754466Z  WARN rustfmt_nightly::test: Default value false used explicitly for format_macro_matchers
2024-07-11T04:27:15.761670Z  WARN rustfmt_nightly::test: Default value false used explicitly for normalize_comments
2024-07-11T04:27:15.765452Z  WARN rustfmt_nightly::test: Default value Never used explicitly for match_arm_leading_pipes
2024-07-11T04:27:15.766551Z  WARN rustfmt_nightly::test: Default value Block used explicitly for imports_indent
2024-07-11T04:27:15.768246Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_macro_bodies
2024-07-11T04:27:15.779770Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.780681Z  WARN rustfmt_nightly::test: Default value 4 used explicitly for tab_spaces
2024-07-11T04:27:15.780685Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.782514Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.784359Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.785312Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.786231Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.787138Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.788491Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.792941Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.794866Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.794870Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.795763Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_line_overflow
2024-07-11T04:27:15.795766Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.795768Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.796635Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.797526Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.800131Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.801093Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.803281Z  WARN rustfmt_nightly::test: Default value false used explicitly for normalize_doc_attributes
2024-07-11T04:27:15.805062Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.806440Z  WARN rustfmt_nightly::test: Default value true used explicitly for combine_control_expr
2024-07-11T04:27:15.806444Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
2024-07-11T04:27:15.807721Z  WARN rustfmt_nightly::test: Default value false used explicitly for hard_tabs
2024-07-11T04:27:15.810238Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_imports
2024-07-11T04:27:15.811118Z  WARN rustfmt_nightly::test: Default value false used explicitly for condense_wildcard_suffixes
2024-07-11T04:27:15.812848Z  WARN rustfmt_nightly::test: Default value false used explicitly for error_on_unformatted
2024-07-11T04:27:15.813731Z  WARN rustfmt_nightly::test: Default value false used explicitly for use_field_init_shorthand
2024-07-11T04:27:15.818439Z  WARN rustfmt_nightly::test: Default value SameLineWhere used explicitly for brace_style
Warning: the `merge_imports` option is deprecated. Use `imports_granularity="Crate"` instead
2024-07-11T04:27:15.842769Z  WARN rustfmt_nightly::test: Default value true used explicitly for merge_derives
2024-07-11T04:27:15.858719Z  WARN rustfmt_nightly::test: Default value One used explicitly for version
2024-07-11T04:27:15.895608Z  WARN rustfmt_nightly::test: Default value Preserve used explicitly for hex_literal_case
Warning: the `fn_args_layout` option is deprecated. Use `fn_params_layout`. instead
2024-07-11T04:27:15.944904Z  WARN rustfmt_nightly::test: Default value true used explicitly for format_macro_bodies
2024-07-11T04:27:15.947915Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_imports
2024-07-11T04:27:15.976438Z  WARN rustfmt_nightly::test: Default value [] used explicitly for skip_macro_invocations
2024-07-11T04:27:15.991039Z  WARN rustfmt_nightly::test: Default value true used explicitly for reorder_modules
2024-07-11T04:27:15.991049Z  WARN rustfmt_nightly::test: Default value SameLineWhere used explicitly for brace_style
2024-07-11T04:27:15.991052Z  WARN rustfmt_nightly::test: Default value 100 used explicitly for max_width
2024-07-11T04:27:15.991054Z  WARN rustfmt_nightly::test: Default value false used explicitly for hard_tabs
2024-07-11T04:27:15.991059Z  WARN rustfmt_nightly::test: Default value 4 used explicitly for tab_spaces
2024-07-11T04:27:15.993025Z  WARN rustfmt_nightly::test: Default value 0 used explicitly for struct_field_align_threshold
2024-07-11T04:27:16.004599Z  WARN rustfmt_nightly::test: Default value Block used explicitly for imports_indent
2024-07-11T04:27:16.004606Z  WARN rustfmt_nightly::test: Default value Block used explicitly for indent_style
thread '<unnamed>' panicked at src/tools/rustfmt/src/test/mod.rs:369:9:
assertion `left == right` failed: 2 idempotent tests failed
  left: 2
 right: 0
```

</details>

--- a/src/tools/rustfmt/tests/source/type.rs
+++ b/src/tools/rustfmt/tests/source/type.rs
@@ -146,8 +146,6 @@ trait T: ~   const  Super {}
 
 const fn not_quite_const<S: ~  const    T>() -> i32 { <S as T>::CONST }
 
-struct S<T:~  const   ?  Sized>(std::marker::PhantomData<T>);
-
 impl ~    const T {}
 
 fn apit(_: impl ~   const T) {}

--- a/src/tools/rustfmt/tests/target/negative-bounds.rs
+++ b/src/tools/rustfmt/tests/target/negative-bounds.rs
@@ -3,9 +3,3 @@ where
     i32: !Copy,
 {
 }
-
-fn maybe_const_negative()
-where
-    i32: ~const !Copy,
-{
-}

--- a/src/tools/rustfmt/tests/target/type.rs
+++ b/src/tools/rustfmt/tests/target/type.rs
@@ -153,8 +153,6 @@ const fn not_quite_const<S: ~const T>() -> i32 {
     <S as T>::CONST
 }
 
-struct S<T: ~const ?Sized>(std::marker::PhantomData<T>);
-
 impl ~const T {}
 
 fn apit(_: impl ~const T) {}

